src/plugins/figure/schema.js

```diff
@@ -35,6 +35,14 @@ function normalizeFigure(change, error) {
         }
         break
 
+    case 'child_type_invalid':
+        if (child.type === 'paragraph') {
+            change.setNodeByKey(child.key, 'figure_caption')
+            break
+        }
+        change.unwrapBlockByKey(child.key)
+        break
```

Comment:
`child_type_invalid` will only trigger if the invalid child is in between two valid children, otherwise `child_unknown` is triggered. In cases where it does trigger it will either split the figure in two, (if invalid child is not a paragraph) where one has no caption and the other is just caption, or (if invalid child is a paragraph) move the existing caption out of the figure (since changing paragraph to `figure_caption` will create two captions and trigger `child_max_invalid`, which will unwrap second (old) caption and convert it to a paragraph). Is this behaviour what you intended?

Reply:
In our editor this normalization is triggered only in specific situations like using `editor.wrapBlock('quotation')` inside `figure_caption` and then it's doing it's job. Other situations are unlikely to happen.

Reply:
If you `wrapBlock('quotation')` a `figure_caption` then it would result in, I believe, a `figure` with two children: a `media` or `figure` followed by a `quotation` (which itself has a single child, a `figure_caption`). In such case `child_unknown` should be triggered.

Reply:
Only `child_type_invalid` is called twice in this case.